plexapi/base.py

```diff
@@ -119,7 +119,7 @@ def _isChildOf(self, **kwargs):
                     See all possible `**kwargs*` in :func:`~plexapi.base.PlexObject.fetchItem`.
         """
         obj = self
-        while obj._parent is not None:
+        while obj._parent is not None and obj._parent() is not None:
             obj = obj._parent()
             if obj._checkAttrs(obj._data, **kwargs):
```

Comment:
I would prefer

```
while obj._parent is not None:
    obj = obj._parent()
    if obj is not None and obj._checkAttrs(obj._data, **kwargs):
        ...
```

Reply:
This will fail on the next iteration of the loop since `obj = None`.

```
AttributeError: 'NoneType' object has no attribute '_parent'
```

We could do
```python
while obj and obj._parent is not None:
    obj = obj._parent()
    if obj is not None and obj._checkAttrs(obj._data, **kwargs):
        ...
```